src/components/OpenSeadragonViewer.js

```diff
@@ -77,7 +80,7 @@ export class OpenSeadragonViewer extends Component {
         tileSources.map((tileSource, i) => this.addTileSource(tileSource, i)),
       ).then(() => {
         if (tileSources[0]) {
-          this.fitBounds(...this.boundsFromTileSources(), true);
+          this.fitBounds(...new CanvasWorld(currentCanvases).worldBounds(), true);
```

Comment:
I realize we are recalculating this everytime we need it. It's not an expensive calculation, so it seems fine to do this way. There could be an oppy in the future to not need this, but I don't this its really necessary.